.ci/Makefile

```diff
@@ -0,0 +1,87 @@
+SHELL = /bin/bash -eo pipefail
+
+AWS_FOLDER = .aws
+AWS_LAMBDA_ZIP_LOCATION = elastic-apm-agent/target
+
+export AWS_FOLDER
+
+dist:
+	@cp $(AWS_LAMBDA_ZIP_LOCATION)/elastic-apm-java-aws-lambda-layer-*.zip $(AWS_LAMBDA_ZIP_LOCATION)/$(ELASTIC_LAYER_NAME).zip
+
+# List all the AWS regions
+get-all-aws-regions:
+	@aws \
+		ec2 \
+		describe-regions \
+		--region us-east-1 \
+		--output json \
+		--no-cli-pager \
+		| jq -r '.Regions[].RegionName' > .regions
+
+# Publish the given LAYER in all the AWS regions
+publish-in-all-aws-regions: validate-layer-name get-all-aws-regions dist
+	@mkdir -p $(AWS_FOLDER)
+	@while read AWS_DEFAULT_REGION; do \
+		echo "publish '$(ELASTIC_LAYER_NAME)' in $${AWS_DEFAULT_REGION}"; \
+		AWS_DEFAULT_REGION="$${AWS_DEFAULT_REGION}" ELASTIC_LAYER_NAME=$(ELASTIC_LAYER_NAME) $(MAKE) publish > $(AWS_FOLDER)/$${AWS_DEFAULT_REGION}; \
+		AWS_DEFAULT_REGION="$${AWS_DEFAULT_REGION}" ELASTIC_LAYER_NAME=$(ELASTIC_LAYER_NAME) $(MAKE) grant-public-layer-access; \
+	done <.regions
+
+# Publish the given LAYER in the given AWS region
+publish: validate-layer-name validate-aws-default-region
+	@aws lambda \
+		--output json \
+		publish-layer-version \
+		--layer-name "$(ELASTIC_LAYER_NAME)" \
+		--description "AWS Lambda Extension Layer for Elastic APM Java" \
```

Comment:
```suggestion
		--description "AWS Lambda Extension Layer for the Elastic APM Java Agent" \
```

Reply:
⬆️ I think for the Java agent layer we should also set the compatible runtimes option when publishing the layer.

Reply:
<img width="827" alt="image" src="https://user-images.githubusercontent.com/2871786/153594351-a45a05a9-79a4-4cba-b516-5efe750452b2.png">

For the record -> [here](https://docs.aws.amazon.com/lambda/latest/dg/lambda-runtimes.html)

Good catch! Thanks 🙇